src/libstd/lib.rs

```diff
@@ -214,10 +214,9 @@
        test(no_crate_inject, attr(deny(warnings))),
        test(attr(allow(dead_code, deprecated, unused_variables, unused_mut))))]
 
-// SNAP 1af31d4
-#![allow(unused_features)]
-// SNAP 1af31d4
-#![allow(unused_attributes)]
+#![cfg_attr(stage0, allow(unused_features))]
+#![cfg_attr(stage0, allow(unused_attributes))]
+#![cfg_attr(stage0, allow(improper_ctypes))]
 
```

Comment:
@alexcrichton why this ? it breaks at least the openbsd build.

`src/libstd/lib.rs` has `drop_in_place` feature, but it used only on some targets. So removing `unused_attributes` will break stage1/stage2 for others.